backend/routes/event_routes.py

```diff
@@ -29,3 +31,13 @@ def create_event():
     new_event = Event.create(**event_json)
 
     return jsonify(new_event.to_dict())
+
+@blueprint.route('/')
+def get_events():
+    events = Event.query.all()
+    all_events = {}
+
+    for event in events:
+        all_events[event.id] = event.to_dict()
+
+    return jsonify(all_events)
```

Comment:
Can replace everything (except the query) with `return jsonify({event.id for event in events})`